control/migrations/0013_jobartefact_order.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 5.2.4 on 2025-08-25 19:18
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('control', '0012_printingproperties_fit_to_page'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='jobartefact',
+            name='order',
+            field=models.IntegerField(default=0),
```

Comment:
See https://github.com/KSIUJ/gutenberg/pull/92/files#r2312025741/. The migration should not set invalid `order` (`document_number`) values.